includes/polling/ports.inc.php

```diff
@@ -744,10 +744,14 @@
         $fields['ifOutErrors_rate'] = $port['ifOutErrors_rate'];
         $fields['ifInOctets_rate'] = $port['ifInOctets_rate'];
         $fields['ifOutOctets_rate'] = $port['ifOutOctets_rate'];
+        $fields['ifInBits'] = $port['stats']['ifInBits_rate'];
```

Comment:
yes i had posted this in grafana community   and opentsdb don't support that https://community.grafana.com/t/how-can-i-use-math-8-with-opentsdb-like-influxdb/1770